src/library_async.js

```diff
@@ -258,14 +276,14 @@ mergeInto(LibraryManager.library, {
       dbg('ASYNCIFY: setDataRewindFunc('+ptr+'), bottomOfCallStack is', bottomOfCallStack, new Error().stack);
 #endif
       var rewindId = Asyncify.getCallStackId(bottomOfCallStack);
-      {{{ makeSetValue('ptr', C_STRUCTS.asyncify_data_s.rewind_id, 'rewindId', 'i32') }}};
+      {{{ makeSetValue('ptr', C_STRUCTS.asyncify_data_s.rewind_id, 'rewindId', MEMORY64 ? 'i64' : 'i32') }}};
```

Comment:
But isn't rewind_id always a 32-bit int:

```
typedef struct asyncify_data_s {                                                 
  void *stack_ptr;     /** Current position in the Asyncify stack (*not* the C stack) */
  void *stack_limit;   /** Where the Asyncify stack ends. */                     
  int rewind_id;       /** Interned ID of the rewind entry point; opaque to application. */
} asyncify_data_t;    
```

Reply:
Let me double check. I'm still working on it, test_emscripten_scan_registers and test_fibers_asyncify don't run correctly

However, generated_struct_info64.json tells that it's 8 bytes
```
"asyncify_data_s": {
            "__size__": 24,
            "rewind_id": 16,
            "stack_limit": 8,
            "stack_ptr": 0
        },
```

Reply:
Changed back to i32 and fixed all other tests. The only one not working now is "test_embind_lib_with_asyncify" but it fixed to 32bit